apps/frontend/pages/404.tsx

```diff
@@ -0,0 +1,39 @@
+import Link from 'next/link'
+import styles from '404.module.css'
+
+export default function Custom404() {
+  return (
+    <div className={styles.wrapper}>
+
+      <h1 className={styles.header}><b>404.</b></h1>
+
+      <a className={styles.properText}>Może szukałeś czegoś innego? Sprawdź to:</a>
+
+      <div className={styles.divlist}>
+
+      <ul className={styles.listmenu}>
+        <li>
+          <Link href="/">
```

Comment:
```suggestion
          <Link href="/" passHref>
```

please add `passHref` and explain why you need to do it, so i'll be sure you've read the docs 😅